README.rst

```diff
@@ -1427,6 +1427,66 @@ List devices
 
     GET /api/v1/controller/device/
 
+**Available filters**
+
+You can filter a list of devices based on their configuration 
+status using the ``status`` (e.g modified, applied, or error).
+
+.. code-block:: text
+
+   GET /api/v1/controller/device/?config__status={status}
+
+You can filter a list of devices based on their configuration backend 
+using the ``backend`` (e.g netjsonconfig.OpenWrt or netjsonconfig.OpenWisp).
+
+.. code-block:: text
+
+   GET /api/v1/controller/device/?config__backend={backend}
+
+You can filter a list of devices based on their 
+organization using the ``organization_id``.
+
+.. code-block:: text
+
+   GET /api/v1/controller/device/?organization={organization_id}
+
+You can filter a list of devices based on their 
+configuration templates using the ``template_id``.
+
+.. code-block:: text
+
+   GET /api/v1/controller/device/?config__templates={template_id}
+
+You can filter a list of devices based on 
+their device group using the ``group_id``.
+
+.. code-block:: text
+
+   GET /api/v1/controller/device/?group={group_id}
+
+You can filter a list of devices that have a device location 
+object using the ``devicelocation`` (eg. true or false).
+
+.. code-block:: text
+
+   GET /api/v1/controller/device/?devicelocation={devicelocation}
```

Comment:
this in the admin is called `with_geo`, can we maintain consitency please?

Reply:
sure